edx_django_utils/cookie_consent/README.rst

```diff
@@ -0,0 +1,25 @@
+Cookie consent
+==============
+
+Utilities for respecting user consent regarding different cookie categories.
+
+"Cookie" is a shorthand here for various types of tracking and storage, but cookies are the main focus.
+
+Configuration
+-------------
+
+Set ``COOKIE_CONSENT_SOURCE`` to the module and name of a subclass of ``ConsentSource``, either one provided by edx-django-utils or from another package. This class will be responsible for answering whether the user has consented to certain categories of cookies. Known values:
+
+- ``edx_django_utils.cookie_consent.OnlyNecessary`` is the default and will only permit strictly necessary cookies.
```

Comment:
Hmm... I suppose that when this lands in Olive, we probably want to start off with `AcceptAll`. But I think we probably want to _eventually_ use `OnlyNecessary` by default, since that simplifies compliance.